concordia/settings_template.py

```diff
@@ -217,6 +217,11 @@
     "loggers": {
         "django": {"handlers": ["file", "stream"], "level": "DEBUG", "propagate": True},
         "celery": {"handlers": ["celery", "stream"], "level": "DEBUG"},
+        "concordia": {
+            "handlers": ["file", "stream"],
+            "level": "DEBUG",
```

Comment:
I'm not sure we want to leave this at DEBUG?

Reply:
Changed to INFO